Telstra.Twins/Attributes/TwinPropertyAttribute.cs

```diff
@@ -26,8 +28,13 @@ public TwinPropertyAttribute(string name)
         }
 
         /// <summary>
-        /// The name of the digital twin property.
+        ///     The name of the digital twin property.
         /// </summary>
         public string Name { get; }

```

Comment:
The compiler is throwing a warning here. 
You can make this `public string Name { get; } = string.Empty`

Reply:
Ah, yes, because I added `#nullable enable` to the top of the class, because I wanted to add the nullable Schema property to the attribute.

Nullable checks are good, because they make you explicitly define if the value can be null or not.

Checking the code, in this case we do want the default value to be null (not string.Empty), as the handling in GetTwinPropertyName() will then default null => p.Name.ToCamelCase(). 

If we set it to Empty as the default, then we would instead get the empty string (which would break a bunch of tests).

Reply:
Nice pickup though. I must of missed the compiler warning.

Rather than default nullable checks in the individual class, we should probably enable it for the entire library (as a separate pull request); turning on "treat warnings as errors" can then help ensure it does not get through again in the future.